pkg/commands/ls.go

```diff
@@ -113,7 +113,14 @@ func mergeStagingComponents(deployed []apis.ComponentMeta, env *types.EnvMeta, i
 					v.Traits[j] = t
 				}
 				aspec.Components[i] = v
+				/* Before any traits is attached to the workload, if ApplicationConfiguration is retrieved through
```

Comment:
Why is it retrieved from OpenAPI?

Reply:
It's retrieved as below https://github.com/oam-dev/kubevela/blob/master/pkg/oam/application.go#L91-L95.
```
var appConfig corev1alpha2.ApplicationConfiguration
		if err := c.Get(ctx, client.ObjectKey{Name: opt.AppName, Namespace: opt.Namespace}, &appConfig); err != nil {
			return appConfigList, err
		}
```